plugins/SegmentEditor/lang/en.json

```diff
@@ -6,6 +6,8 @@
         "AreYouSureDeleteSegment": "Are you sure you want to delete this segment?",
         "AutoArchivePreProcessed": "segmented reports are pre-processed (faster, requires cron)",
         "AutoArchiveRealTime": "segmented reports are processed in real time",
+        "ChangingSegmentDefinitionConfirmationNotProcessedOnRequest": "You're about to change the segment definition. Your analytics reports for this new segment won't be available until the reports are re-processed. It may take a few hours for reports data to show for this segment. Proceed anyway?",
```

Comment:
Sure we could do that. But maybe it's better to have complete messages as one string. That prevents from having only parts of it translated, which would look very ugly in the UI I think.